go/libraries/doltcore/schema/typeinfo/linestring.go

```diff
@@ -214,3 +222,18 @@ func linestringTypeConverter(ctx context.Context, src *linestringType, destTi Ty
 		return nil, false, UnhandledTypeConversion.New(src.String(), destTi.String())
 	}
 }
+
+func CreateLinestringTypeFromParams(params map[string]string) (TypeInfo, error) {
+	sridVal, err := strconv.ParseInt(params["SRID"], 10, 32)
+	if err != nil {
+		return nil, err
+	}
+
+	dSRID, err := strconv.ParseBool(params["DefinedSRID"])
+	if err != nil {
+		return nil, err
+	}
+
+	LinestringType = &linestringType{sqlLinestringType: sql.LinestringType{SRID: uint32(sridVal), DefinedSRID: dSRID}}
```

Comment:
Same comment here as with `Geometry` – I'm not sure we want to cache the last instantiated Linestring TypeInfo and share it globally. Let me know if I'm missing something.